internlm/model/metrics.py

```diff
@@ -155,7 +155,7 @@ def update(self, logits, labels, type_ids=None):
 
             acc = corrects.sum()
             torch.distributed.all_reduce(acc, op=torch.distributed.ReduceOp.SUM, group=self.tp_pg)
-            torch.npu.synchronize()
+            internlm_accelerator.synchronize()
```

Comment:
internlm_accelerator = get_accelerator() has already been defined, so no need to use get_accelerator()?